gui/threadresult.cpp

```diff
@@ -58,14 +58,15 @@ void ThreadResult::reportErr(const ErrorMessage &msg)
         emit debugError(item);
 }
 
-QString ThreadResult::getNextFile()
+void ThreadResult::getNextFile(const FileWithDetails*& file)
```

Comment:
Can we return the pointer instead of using outparam?

Reply:
I tried to keep it in line with the existing code here and the duplicated code in `ThreadData` from `threadexecutor.cpp`.